Open Judge System/Services/OJS.Services/Cache/CacheItemsProviderService.cs

```diff
@@ -180,16 +184,56 @@ public IEnumerable<HomeContestViewModel> GetActiveContests()
                     .ToList(),
                 TimeSpan.FromSeconds(CacheConstants.OneHourInSeconds));
 
+        public IDictionary<int, ParticipantsCountCacheModel> GetParticipantsCountForContestsInCategoryPage(
```

Comment:
I think it will be good to make the time a readonly variable. Also for the other cache keys we are using this pattern "Something1_something2" etc .. Maybe we can change it here to follow same pattern everywhere

Reply:
Moved time to readonly and cached it also.

For Redis `:` is the standardized convention for a delimiter. This will also make a folder structure of the keys, when using redis exporer to view your keys, whereas with `_` all the keys (which can be thousands) will be in a single folder. We should fix all the places where `_` is used instead, and there is an issue for that: https://github.com/SoftUni-Internal/exam-systems-issues/issues/843